handbook/communication/index.md

```diff
@@ -16,7 +16,8 @@ These places are the source of truth for information at Sourcegraph. Information
 - [Project roadmap](https://docs.google.com/document/d/1cBsE9801DcBF9chZyMnxRdolqM_1c2pPyGQz15QAvYI/edit#)
 - [Issue tracker](https://github.com/sourcegraph/sourcegraph)
 - [productboard](https://sourcegraph.productboard.com/)
-- [HubSpot](https://app.hubspot.com)
+- [HubSpot](https://app.hubspot.com) (contacts only)
```

Comment:
What does "(contacts only)" mean?

Reply:
As opposed to deals and companies. Will make a clarification